alica_ros_turtlesim/include/alica_ros_turtlesim/world_model.hpp

```diff
@@ -0,0 +1,38 @@
+#ifndef ALICA_TURTLE_WORLD_MODEL_WORLD_MODEL_HPP
+#define ALICA_TURTLE_WORLD_MODEL_WORLD_MODEL_HPP
+
+#include <ros/ros.h>
+#include <engine/AlicaEngine.h>
+#include <std_msgs/Empty.h>
+#include "alica_ros_turtlesim/turtle.hpp"
+
+namespace turtlesim {
+/*
+        ALICATurtleWorldModel
+        - WorldModel for ALICA ros turtlesim which interface between ALICA and ROS.
+        - A few class is static since one robot has one world model
+        - ROS:
+                - Subscribe: t/init
+*/
+class ALICATurtleWorldModel {
+public:
+    static ALICATurtleWorldModel* get();                              // return instance
+    static void init(ros::NodeHandle& nh, ros::NodeHandle& priv_nh);  // create instance
+    static void del();
+    bool get_init() const { return _init_trigger; };
```

Comment:
alica naming convention is camel case. Ex: getInit()
@hendrik-skubch can samples be exempted?